streams/src/main/java/org/apache/kafka/streams/kstream/KTable.java

```diff
@@ -70,7 +70,7 @@
 
     /**
      * Create a new {@code KTable} that consists of all records of this {@code KTable} which satisfy the given
-     * predicate.
+     * predicate, with default serializers, deserializers and internal state storage.
```

Comment:
Nit: add Oxford comma: `deserializers, and`

Nit: `internal state storage` -> `state store configuration`